smoke/ip6_forward_test.sh

```diff
@@ -11,8 +11,6 @@ grcli add interface port $p1 devargs net_tap0,iface=$p1 mac d2:f0:0c:ba:a4:11
 grcli add interface port $p2 devargs net_tap1,iface=$p2 mac d2:f0:0c:ba:a4:12
 grcli add ip6 address fd00:ba4:1::1/64 iface $p1
 grcli add ip6 address fd00:ba4:2::1/64 iface $p2
-grcli add ip6 nexthop fd00:ba4:1::2 mac d2:ad:ca:ca:a4:11 iface $p1
```

Comment:
And if you remove the DAD comment from previous patch, you can add it in this patch.